src/graphql/admin/types/object/account.ts

```diff
@@ -36,7 +38,7 @@ const Account: GraphQLObjectType<Account> = GT.Object<Account>({
 
       type: GT.NonNull(GraphQLUser),
       resolve: async (source) => {
-        const user = await Users.getUser(source.ownerId)
+        const user = await UsersRepository().findById(source.kratosUserId)
```

Comment:
@dolcalmi does this have any implication for open telemetry?

Reply:
Yes, this removes the span wrapping we do in `src/app/index.ts`. It's probably ok though, because any inner tracing on this repository method should be caught by the parent span here I think.

Reply:
also graphql layer should only access app layer